### ISSUE TITLE
Add code action for the table related diagnostics

### ISSUE THREAD

**Issue by nadeeshaan** [ballerina-platform/ballerina-lang#31185]
**Description:**
Consider the following capture.

<img width="1144" alt="Screenshot 2021-06-15 at 17 03 52" src="https://user-images.githubusercontent.com/1329674/122045817-cb866500-cdfb-11eb-8549-58c88d29fbcf.png">

Provide a code action to add the particular field.

**Comment by mohanvive** [comment]
Additionally, completion suggestion with field names?

**Comment by nadeeshaan** [comment]
> Additionally, completion suggestion with field names?

@mohanvive yes. Let's address this with the completions of the table constructor. +1
